### PR TITLE
Fix command name mention in help text.

### PR DESCRIPTION
**Pull request by Veebers**
A simple fix in some help text, the command update-clouds was misspelled.

### CODE REVIEW

File: cmd/juju/cloud/list.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ var listCloudsDoc = "" +
 	"Provided information includes 'cloud' (as understood by Juju), cloud\n" +
 	"'type', and cloud 'regions'.\n" +
 	"The listing will consist of public clouds and any custom clouds made\n" +
-	"available through the `juju add-cloud` command. The former can be updated\n" +
+	"available through the `juju add-clouds` command. The former can be updated\n" +
```
[the remark this answers is not in the document]
Comment (anastasiamac) [reply cmd/juju/cloud/list.go]:
```
$ juju help commands
actions                      List actions defined for an application.
add-cloud                    Adds a user-defined cloud to Juju from among known cloud types.
add-credential               Adds or replaces credentials for a cloud, stored locally on this client.
add-k8s                      Adds a k8s endpoint and credential to Juju.
add-machine                  Start a new, empty machine and optionally a container, or add a container to a machine.
add-model                    Adds a hosted model.
add-relation                 Add a relation between two application endpoints.
add-space                    Add a new network space.
add-ssh-key                  Adds a public SSH key to a model.
add-storage                  Adds unit storage dynamically.
add-subnet                   Add an existing subnet to Juju.
add-unit                     Adds one or more units to a deployed application.
add-user                     Adds a Juju user to a controller.
agree                        Agree to terms.
agreements                   List user's agreements.
attach                       Alias for 'attach-resource'.
attach-resource              Upload a file as a resource for an application.
attach-storage               Attaches existing storage to a unit.
autoload-credentials         Attempts to automatically add or replace credentials for a cloud.
....
```

**Comment by babbageclunk** [comment]
But you might want to fix the description too for full stealthiness.

**Comment by Veebers** [comment]
Why would I need stealth, this PR was correct on the first try, a single commit no fix needed :grimacing:

**Comment by Veebers** [comment]
$$merge$$

**Pull request merged**